src/resolve-dependency.ts

```diff
@@ -95,15 +107,19 @@ function getExportsTarget (exports, conditions, cjsResolve) {
       }
     }
   }
-  else if (exports === null) {
-    return exports;
-  }
+
+  return null;
```

Comment:
Is it okay to return null here or should it be undefined?

Reply:
It needs to be `undefined` not `null`. They have different meanings for this function.

Reply:
Thanks, updated to `return undefined` 👍